### PR TITLE
Remove Frontend JSON API

### PR DESCRIPTION
**Pull request by emmabeynon**
In order to remove Frontend's JSON API, we have deleted the ApiRedirectable module and references to it.

https://trello.com/c/7IRxAVoJ/688-retire-frontend-s-json-api

Paired with @cbaines

### CODE REVIEW

**Comment by davidbasalla** [comment]
There's [a line in the routes.rb file](https://github.com/alphagov/frontend/blob/master/config/routes.rb#L26) that looks like it could be removed too?

**Comment by emmabeynon** [comment]
> There's a line in the routes.rb file that looks like it could be removed too?

Good spot - now removed 🎉

File: lib/core-ext/openstruct.rb [hunk 1]
```diff
@@ -1,5 +0,0 @@
-class OpenStruct
```
Comment (whoojemaflip) [inline lib/core-ext/openstruct.rb]: this is freaking me out a little. Any idea what it was for?

File: app/presenters/publication_presenter.rb [hunk 2]
```diff
@@ -48,10 +48,15 @@ def updated_at
     DateTime.parse(date) if date
   end
 
-  def to_json
-    {
-      places: places
-    }.to_json
+  def video_embed_url
```
Comment (whoojemaflip) [inline app/presenters/publication_presenter.rb]: not convinced that this belongs in this PR

File: lib/core-ext/openstruct.rb (hunk 1 above)
Comment (emmabeynon) [inline lib/core-ext/openstruct.rb]: Looking at the commit history it was created for use in the root controller, which no longer exists.  We couldn't find any other use of it in the rest of the app.

File: app/presenters/publication_presenter.rb (hunk 2 above)
Comment (emmabeynon) [inline app/presenters/publication_presenter.rb]: We can move this into a separate PR if you'd like.

Comment (emmabeynon) [inline app/presenters/publication_presenter.rb]: Sorry at first I thought you were referring to the `to_json` method, now I realise it's about the `video_embed_url` method.  This has been removed in another PR, which is why I think this looks like it's being added.  Now removed it again 👍

**Comment by brenetic** [comment]
do we still need the `help.json`,  `licence-generic.json` and the `pay-bear-tax.json` fixtures?

**Comment by emmabeynon** [comment]
@brenetic 
> do we still need the help.json, licence-generic.json and the pay-bear-tax.json fixtures?

I tried deleting `help.json` but it broke tests related to rendering the help page.  `pay-bear-tax.json` and `licence-generic.json` did not break tests however so those are now deleted 👍
